integration_tests/schemas/update.sh

```diff
@@ -29,7 +29,7 @@ rm ${stac_tag}.tar.gz
 rm -rf "schemas.stacspec.org/${stac_version}"
 mv ${subfolder} "schemas.stacspec.org/${stac_version}"
 rm -rf "stac/${stac_version}"
-ln -s "$(pwd)/schemas.stacspec.org/${stac_version}" "$(pwd)/stac"
+ln -s "../schemas.stacspec.org/${stac_version}" "stac/"
```

Comment:
This is the sort of unintuitive line it'd be worth adding a comment to. I can imagine someone tripping over it in the future.

Reply:
Done!